include/lorawan/lorawan.h

```diff
@@ -104,6 +104,11 @@ struct lorawan_join_config {
 	enum lorawan_act_type mode;
 };
 
+struct lorawan_nvm_crypto_context {
+	/** Parameters are internal implementation details of loramac-node */
+	uint8_t _raw[52];
```

Comment:
Because the struct is an implementation detail of loramac-node, and isn't visible to applications with the current cmake setup. Duplicating the fields is more error prone than just duplicating and verifying the size. The majority of the fields are of no interest to applications anyway.